traffic_monitor/health/cache.go

```diff
@@ -53,14 +53,44 @@ func GetVitals(newResult *cache.Result, prevResult *cache.Result, mc *tc.Traffic
 		return
 	}
 
+	if mc == nil {
+		log.Errorf("mc TrafficMonitorConfigMap cannot be nil")
```

Comment:
The word `mc` might not make much sense in the logs.

Reply:
Will remove.

Reply:
Done